project2/style.css

```
@@ -0,0 +1,66 @@
+.title {
+    font-family: 'ZCOOL QingKe HuangYou', cursive;
+    color: white;
+    font-size: 19pt;
+    line-height: 50px;
+    margin-left: 8px;
+    padding-right: 20px;
+
```

More your inline styles here instead:
```suggestion
  width: 360px;
```